src/utils/date-helpers.ts

```diff
@@ -55,3 +56,15 @@ export const toDateString = (date: Date | string | number) => {
 
   return dateObject.toLocaleDateString('nb-NO', { year: 'numeric', month: '2-digit', day: '2-digit' });
 };
+
+export const toDateStringWithTime = (date: Date | string | number) => {
```

Comment:
kan eventuelt sende inn `t` som argument i stedet for å importere den statisk.
Fordelen med løsningen er per nå er at det blir litt mindre kode, ulempen er at bruker må refreshe siden for å få riktig oversettelse når man bytter språk

Reply:
Tenker det er såpass niche at folk endrer språk mens de er inne og fikser tickets, og det er uansett ganske åpenbart hva det er snakk om 🤔